docs/bundles/SyliusFixturesBundle/builtin_listeners.rst

```diff
@@ -0,0 +1,80 @@
+Built-in listeners
+==================
+
+**SyliusFixturesBundle** comes bundled with a few useful listeners.
+
+Logger (``logger``)
+-------------------
+
+Provides output during running ``sylius:fixtures:load`` command.
+
+.. code-block:: bash
+
+    # Without logger
+
+    $ app/console sylius:fixtures:load my_suite
+    $ _
+
+    # With logger
+
+    $ app/console sylius:fixtures:load my_suite
+    Running suite "my_suite"...
+    Running fixture "country"...
+    Running fixture "locale"...
+    Running fixture "currency"...
+    $ _
+
+Has no configuration options, can be enabled by:
+
+.. code-block:: yaml
+
+    sylius_fixtures:
+        suites:
+            my_suite:
+                listeners:
+                    logger: ~
+
+ORM Purger (``orm_purger``)
+---------------------------
+
+Purges the relational database. Uses ``delete`` purge mode and the default entity manager if not configured otherwise.
+
+Configuration options:
+
+    - ``purge_mode`` - sets how databse is purged, available values: ``delete`` (default), ``truncate``
+    - ``managers`` - array of entity managers names used to purge the database, ``[null]`` by default
+
+Example configuration:
+
+.. code-block:: yaml
+
+    sylius_fixtures:
+        suites:
+            my_suite:
+                listeners:
+                    orm_purger:
+                        purge_mode: truncate
+                        managers:
+                            - custom_manager
+
+PHPCR / MongoDB Purger (``phpcr_purger`` / ``mongodb_purger``)
+--------------------------------------------------------------
+
+Purges the document database. Uses the default document manager if not configured otherwise.
+
+Configuration options:
+
+    - ``managers`` - array of document managers names used to purge the database, ``[null]`` by default
```

Comment:
`... an array of entity managers' names used to purge the documents database ...`